dit/dit_exporter.py

```diff
@@ -199,10 +200,11 @@ def _phrase(description, value):
 
             if time_spent:
                 string += "%s %s. " % (_ce('Spent'), td2str(time_spent))
+            string += "\n  %s %s." % (_ce('First clocked in at'), dt2str(first_entry['in']))
             if log['out']:
-                string += "%s %s." % (_ce('Clocked out at'), dt2str(log['out']))
+                string += "\n  %s %s." % (_ce('Clocked out at'), dt2str(log['out']))
             else:
-                string += "%s %s." % (_cf('Clocked in at'), dt2str(log['in']))
+                string += "\n  %s %s." % (_cf('Clocked in at'), dt2str(log['in']))
```

Comment:
Instead of manually breaking lines, I think it would be better to call `_write`.